### ISSUE TITLE
Text and Links Jumping While Writing

### ISSUE THREAD

**Issue by nesoor** [the-djmaze/snappymail#1004]
**Describe the bug**
When writing an email, the text jumps up and links (URLs) skip a line, even when copying and pasting a clean version of the email with no HTML or other code. This occurs when trying to place a link on the same line as the text.

**To Reproduce**
1. Set up an email signature.
2. Write a text and try to copy and paste an email address or link, attempting to place it on the same line as the text.

**Expected behavior**
The link should remain on the same line as the text and not jump up and down.

**Screenshots**
Please see attached video content.

**Please complete the following information:**
- Chrome: Version 110.0.5481.178 (Official Build) (64-bit)
- PHP version: 8.3
- SnappyMail Version: 2.26.4

https://user-images.githubusercontent.com/19656499/222990506-4df67021-f9f3-4770-8bb7-1cf0902475d0.mp4

**Comment by andreasdirscherl** [comment]
+1
Same here. 

I noticed that it starts to "misbehave" after one pastes something into the message body. Even when it is plain text, no formatting. 
I took a look at the HTML, and for some reason, all spaces from source text get transformed to &nbsp; and then the jumping occurs. Because it suddenly is one long junk of text.
This even happens when I cut a part of the message I'm just writing and paste it elsewhere. Again, all &nbsp;
Very weird.

**Comment by nesoor** [comment]
> +1 Same here.
> 
> I noticed that it starts to "misbehave" after one pastes something into the message body. Even when it is plain text, no formatting. I took a look at the HTML, and for some reason, all spaces from source text get transformed to   and then the jumping occurs. Because it suddenly is one long junk of text. This even happens when I cut a part of the message I'm just writing and paste it elsewhere. Again, all   Very weird.

After removing the white space beneath the text "Signature" and beginning to write or copy-paste, everything appears to function correctly.

**Comment by andreasdirscherl** [comment]
Mmh.... not for me. The &nbsp;  issue is what's causing it for me. 
I started a brand-new message with nothing in it, no signature,  no nothing. All fine until I paste something. Anything. Then the non-breaking spaces are introduced.

**Comment by nesoor** [comment]
> Mmh.... not for me. The   issue is what's causing it for me. I started a brand-new message with nothing in it, no signature, no nothing. All fine until I paste something. Anything. Then the non-breaking spaces are introduced.

I have the exact same issue, except that when I insert a signature, it creates an automatic white space below it. If I remove that white space first and then start writing, everything works fine.

**Comment by andreasdirscherl** [comment]
Sorry, haven't seen this until today. I'm afraid I still experience the jumping text. Sorry. 
There's also a similar new issue [#1054](https://github.com/the-djmaze/snappymail/issues/1054#issue-1639397176).
It seems they are related?

**Comment by andreasdirscherl** [comment]
Sorry, haven't seen this until today. I'm afraid I still experience the jumping text. Sorry. 
There's also a similar new issue [#1054](https://github.com/the-djmaze/snappymail/issues/1054#issue-1639397176).
It seems they are related?

**Comment by nesoor** [comment]
> Sorry, haven't seen this until today. I'm afraid I still experience the jumping text. Sorry. There's also a similar new issue [#1054](https://github.com/the-djmaze/snappymail/issues/1054#issue-1639397176). It seems they are related?

Have you downloaded the latest version?
The latest version fixed it for me.

**Comment by andreasdirscherl** [comment]
Thanks!
Yes, I am on 2.27.2.

My issue is a little different, as it relates to introducing the `&nbsp;` on copy and paste...

**Comment by the-djmaze** [comment]
I've tested to see what is happening:

1. Paste link
![afbeelding](https://user-images.githubusercontent.com/3752035/228215718-00aee022-fd75-47de-83ae-063dfd08b6d4.png)

2. Backspace
![afbeelding](https://user-images.githubusercontent.com/3752035/228215920-bfa6d337-a43e-456b-8733-9600d1618b77.png)

I've traced back this issue to the solution for #962

It seems i accidentally used `container.children` and not `container.childNodes`.
The PR i made at Squire does use `container.childNodes` and does not have the problem.
